spacy/errors.py

```diff
@@ -934,6 +934,7 @@ class Errors(metaclass=ErrorsWithCodes):
     E1041 = ("Expected a string, Doc, or bytes as input, but got: {type}")
     E1042 = ("Function was called with `{arg1}`={arg1_values} and "
              "`{arg2}`={arg2_values} but these arguments are conflicting.")
+    E1043 = ("Expected None or a value in range [0, 1] for abstention threshold, but got {value}.")
```

Comment:
Maybe instead of "abstention threshold" we can make this "EntityLinker threshold" but actually give the component name to `E1043` as a parameter? (kind of like how `W088` is used)